sqlalchemy_ibmi/base.py

```diff
@@ -130,7 +130,7 @@ class User(Base):
 """
 import datetime
 import re
-from distutils import util
+from distutils import util as du_util
```

Comment:
Since we're only using strtobool, we could be a bit cleaner by importing just that function:

```suggestion
from distutils.util import strtobool
```

Then drop the qualifier from the strtobool calls later on.

Reply:
Changed per your guidance.  Thanks!